src/Mod/Fem/femsolver/calculix/tasks.py

```diff
@@ -69,6 +69,10 @@ def run(self):
             c.beam_sections, c.shell_thicknesses, c.fluid_sections,
             self.solver.AnalysisType, self.directory)
         path = w.write_calculix_input_file()
+        if path is not None:
```

Comment:
perhaps a comment should precede this section of the code like:  
`# report ot user if task succeeded`?

Reply:
https://github.com/FreeCAD/FreeCAD/pull/1210/commits/c6740dc89fe802861a25947fa8633ae55ee268f9